packages/block-editor/src/components/off-canvas-editor/block-contents.js

```diff
@@ -122,7 +122,10 @@ const ListViewBlockContents = forwardRef(
 						} }
 					/>
 				) }
-				<BlockDraggable clientIds={ draggableClientIds }>
+				<BlockDraggable
+					clientIds={ draggableClientIds }
+					context="list-view"
```

Comment:
Tell the draggable it's in the list view context.

Reply:
My gut reaction is that the block draggable should know about itself and only allow dropping onto itself by default, and we should not need a prop. Maybe a sort of id can be generated and then on drop we check that?

Reply:
I see what you're suggesting.

However, as I understand it `BlockDraggable` is a component which allows a set of blocks to be dragged but doesn't itself define the dropzone.